docs/content/concepts/graphql-rpc.mdx

```diff
@@ -231,11 +240,11 @@ This property requires that cursors that are used together (for example when sup
 
 #### Available range
 
-The GraphQL service does not support consistent pagination for arbitrarily old cursors. A cursor can grow stale, if the checkpoint it is from is no longer in the **available range**. You can query the upper- and lower-bounds of that range as follows:
+The GraphQL service does not support consistent pagination for arbitrarily old cursors. A cursor can grow stale, if the checkpoint it is from is no longer in the **available range**. Data outside of the available range can also be pruned away. This range might be different depending on the specific query and filters used. You can query the upper- and lower-bounds of that range for your query as follows:
```

Comment:
This passage is a bit too specific to retention for the live object set queries, we should generalise it (and maybe rename the section to "Retention" as well). Feel free to rephrase, but this is what I came up with:

```suggestion
Different queries are subject to their own retention policies. For example, Live object set queries only work for recent checkpoints (measured in minutes or hours), and transaction pagination may only work going back to a certain checkpoint (measured in weeks or months) with different retentions supported based on the filters applied. Data outside these ranges will be pruned away. You can query the upper- and lower-bounds of that range for your query as follows:
```